### ISSUE TITLE
Screen titles and headings not identified [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#217]
The following screen titles are not announced as headings by TalkBack.

“Help stop the spread …” text on the Onboarding Introduction screen
“Set Up Closeness Sensing …” text on the Onboarding ENS Intro screen
“How it works” text on the How it works screens
“Closeness Sensing and COVID Alerts are active” text on the Onboarding Confirm Success screen
“NY COVID Data” text on the NY COVID Data screen

Subheadings on screens are not marked up as headings.

![onboarding ENS intro - subheadings](https://user-images.githubusercontent.com/7034772/91994669-db1d3d00-ed04-11ea-960b-a8d69348d0d7.png)

There are likely more instances that will found upon further testing. I would categorize this issue as low severity.

**Comment by floridemai** [comment]
@jonnyjames This has now been addressed. Not that I've chosen to group the permissions so the whole text is read at once. Please re-open if that's not correct.

**Comment by jonnyjames** [comment]
@floridemai I think it's fine and it works. I am closing this issue.
